plugins.json

```
@@ -1426,5 +1426,17 @@
         "UrlDownload": "https://github.com/DiekoMA/Flow.Launcher.Plugin.Anilist/releases/download/v1.0.0/Anilist.zip",
         "DateAdded": "2023-04-16T23:07:57Z",
         "LatestReleaseDate": "2023-04-03T16:14:34Z"
+    },
+    {
+        "ID":"5DBA462B9AB77AC5DC158EB5047367F0",
+        "Name":"RemoveUSB",
+        "Description":"Removes USB drives",
+        "Author":"Jonas A. Wendorf",
+        "Version":"1.0.0",
+        "Language":"python",
+        "Website": "https://github.com/jonasw234/FlowLauncher.Plugin.RemoveUSB",
+        "IcoPath": "https://cdn.statically.io/gh/jonasw234/FlowLauncher.Plugin.RemoveUSB/main/icon/usb_stick.png",
+        "UrlSourceCode": "https://github.com/jonasw234/FlowLauncher.Plugin.RemoveUSB",
+        "UrlDownload": "https://github.com/jonasw234/FlowLauncher.Plugin.RemoveUSB/releases/download/1.0.0/FlowLauncher.Plugin.RemoveUSB.zip"
```

```suggestion
        "UrlDownload": "https://github.com/jonasw234/FlowLauncher.Plugin.RemoveUSB/releases/download/v1.0.0/FlowLauncher.Plugin.RemoveUSB.zip"
```
Test is unable to find release download.
New versioning has "v" prepended